CONTRIBUTING.md

```diff
@@ -62,6 +61,16 @@ After that, push your changes to your fork.
 git push origin [branch_name]
 ```
 
+Picking up new issues in parallel or after your pull request has been merged in the main BitcoinSource repo <--- Always work on a new branch *** but also keep your fork and local origin/master in sync with upstream/master
+
+git checkout master
+git pull upstream master (Or git fetch upstream master; git merge upstream/master)
+git push
+git checkout -b new-issue
```

Comment:
This entire sequence is already covered in this document. I don't think that we need to repeat it.

Reply:
Hi @slashrsm the part that covers it talks abt - "Update your local master branch" but doesn't cover updating your fork.

This is to update the fork. Unless that is also covered somewhere.